src/System.IO.FileSystem/src/System/IO/FileStatus.Unix.cs

```diff
@@ -177,7 +177,7 @@ internal DateTimeOffset GetLastAccessTime(ReadOnlySpan<char> path, bool continue
             return UnixTimeToDateTimeOffset(_fileStatus.ATime, _fileStatus.ATimeNsec);
         }
 
-        internal void SetLastAccessTime(string path, DateTimeOffset time) => SetAccessWriteTimes(path, time, null);
+        internal void SetLastAccessTime(string path, DateTimeOffset time) => SetAccessWriteTimes(path, time, true);
```

Comment:
Usually in boolean parameters we use named parameters to understand what is that value for. I.e:

`SetAccessWriteTimes(path, time, isAccessTime: true);`